### PR TITLE
Feat/port ft mars 65

### PR DESCRIPTION
**Pull request by wonderbeel**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* Add new keyboard - ft/mars65 (QMK and Via)

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/ft/mars65/config.h [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+// /*== or choose animations ==*/
+//     #define RGBLIGHT_EFFECT_BREATHING
+//     #define RGBLIGHT_EFFECT_RAINBOW_MOOD
+//     #define RGBLIGHT_EFFECT_RAINBOW_SWIRL
+//     #define RGBLIGHT_EFFECT_SNAKE
+//     #define RGBLIGHT_EFFECT_KNIGHT
+//     #define RGBLIGHT_EFFECT_CHRISTMAS
+//     #define RGBLIGHT_EFFECT_STATIC_GRADIENT
+//     #define RGBLIGHT_EFFECT_RGB_TEST
+//     #define RGBLIGHT_EFFECT_ALTERNATING
```
Comment (drashna) [inline keyboards/ft/mars65/config.h:50]: Could you explicitly define which animations are enabled? 
```suggestion
#define RGBLIGHT_EFFECT_BREATHING
#define RGBLIGHT_EFFECT_RAINBOW_MOOD
#define RGBLIGHT_EFFECT_RAINBOW_SWIRL
#define RGBLIGHT_EFFECT_SNAKE
#define RGBLIGHT_EFFECT_KNIGHT
#define RGBLIGHT_EFFECT_CHRISTMAS
#define RGBLIGHT_EFFECT_STATIC_GRADIENT
#define RGBLIGHT_EFFECT_RGB_TEST
#define RGBLIGHT_EFFECT_ALTERNATING
#define RGBLIGHT_EFFECT_TWINKLE
```

File: keyboards/ft/mars65/readme.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Mars 6.5
+
+65% Keyboard with in switch backlight and RGB Underglow. 
+
+![Mars 6.5 image](https://i.imgur.com/cg5qADr.jpg)
```
Comment (drashna) [inline keyboards/ft/mars65/readme.md:5]:
```suggestion
![Mars 6.5 image](https://i.imgur.com/cg5qADrl.jpg)
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Comment by wonderbeel** [comment]
@drashna Thanks for the review, I commited your suggestions :+1: .

**Review by drashna — APPROVED** [review]

File: keyboards/ft/mars65/readme.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# Mars 6.5
+
+65% Keyboard with in switch backlight and RGB Underglow. 
+
+![Mars 6.5 image](https://i.imgur.com/cg5qADrl.jpg)
```
Comment (keyboard-magpie) [inline keyboards/ft/mars65/readme.md:5]: Imgur link no longer works, when you replace it you can append one of the following sizes to make it more appropriately sized:

```
s =   90×  90 = Small Square (as seen in the example above)
b =  160× 160 = Big Square 
t =  160× 160 = Small Thumbnail 
m =  320× 320 = Medium Thumbnail 
l =  640× 640 = Large Thumbnail
h = 1024×1024 = Huge Thumbnail
```

I.e. for the original link `https://i.imgur.com/cg5qADrlh.jpg`

**Review by keyboard-magpie — CHANGES_REQUESTED** [review]

File: keyboards/ft/mars65/readme.md (hunk 3 above)
Comment (wonderbeel) [reply keyboards/ft/mars65/readme.md]: > `https://i.imgur.com/cg5qADrlh.jpg`
Doesn't seem to work, checked with curl too to be 100% sure that it wasn't a cache error and it returns a 302, I replaced it with `https://i.imgur.com/cg5qADrh.jpg` instead that also double checking with curl returns a 200.

Comment (keyboard-magpie) [reply keyboards/ft/mars65/readme.md]: No worries, I was maybe being overzealous as your image actually is appropriately sized already, sorry!

**Review by keyboard-magpie — APPROVED** [review]

**Pull request merged**